include/extractor/turn_instructions.hpp

```diff
@@ -31,37 +36,98 @@ enum class TurnInstruction : unsigned char
     AccessRestrictionPenalty = 129
 };
 
-// Translates between angles and their human-friendly directional representation
-inline TurnInstruction getTurnDirection(const double angle)
+inline TurnInstruction shiftTurnToLeft(TurnInstruction turn)
```

Comment:
The next 100 lines or so look like a small state machine to me. Are there better ways of expressing those?